apps/registration/src/components/ImageInputBox.tsx

```diff
@@ -0,0 +1,93 @@
+import { useRef } from 'preact/compat';
+
+export const ImageInputBox = ({
```

Comment:
A little bit of change krub, please use `export default` so we don't need to import it as a module when we use it krub.

thanks